pkg/ibmcsidriver/ibm_csi_driver.go

```diff
@@ -103,6 +103,7 @@ func (icDriver *IBMCSIDriver) SetupIBMCSIDriver(provider cloudProvider.CloudProv
 	icDriver.ids = NewIdentityServer(icDriver)
 	icDriver.ns = NewNodeServer(icDriver, mounter, statsUtil, metadata)
```

Comment:
Updating this function to take the volumeAttachLimit will take care of addressing the above comment